Game/Source/GDKTestGyms/Private/TestGymsReplicationGraph.cpp

```diff
@@ -757,6 +917,87 @@ void UTestGymsReplicationGraphNode_GlobalViewTarget::LogNode(FReplicationGraphDe
 	DebugInfo.PopIndent();
 }
 
+
+void UTestGymsReplicationGraphNode_NearestActors::NotifyAddNetworkActor(const FNewReplicatedActorInfo& ActorInfo)
+{
+	ReplicationActorList.Add(ActorInfo.Actor);
+}
+
+bool UTestGymsReplicationGraphNode_NearestActors::NotifyRemoveNetworkActor(const FNewReplicatedActorInfo& ActorInfo, bool bWarnIfNotFound /*= true*/)
```

Comment:
Maybe we should warn if not found actor to remove?

Reply:
I don't think that's necessary, we return if the remove was successful or not, so the user of the node could warn if necessary.

Reply:
Seems a bit awkward to ignore part of the API surface of this virtual function though? I would be pretty surprised if I called this on a base class pointer and it did not warn.

Reply:
Oh you guys are right, I completely missed the `bWarnIfNotFound` variable - will fix.